content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -123,12 +123,12 @@ of provisioning.
 Bootstrap tokens are a simpler and more easily managed method to authenticate kubelets, and do not require any additional flags when starting kube-apiserver. 
 Using bootstrap tokens is currently __beta__ as of Kubernetes version 1.12.
 
-Whichever method you choose, the requirement is that the kubelet be able to authenticate as a user in the `system:bootstrappers` group. This group
-serves two purposes:
+Whichever method you choose, the requirement is that the kubelet be able to authenticate as a user with the rights to:
 
-1. Will be granted rights to create and retrieve CSRs.
-2. Used in the controller-manager configuration to scope
-approvals in the default approval controller, if automatic approval is enabled. 
+1. create and retrieve CSRs
+2. Be recognized by the controller-manager configuration to scope approvals in the default approval controller, if automatic approval is enabled. 
+
+A kubelet authenticating usinng bootstrap tokens is recognized as a group `system:bootstrappers`, which is the standard method to use.
```

Comment:
s/usinng/using/

"is authenticated as a user in the `system:bootstrappers` group"

Reply:
Want to guess which letter is misbehaving in my MacBookPro keyboard this week?

Sure.